StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Controllers/PayWithLinkViewController-WalletViewController.swift

```diff
@@ -55,7 +55,10 @@ extension PayWithLinkViewController {
             callToAction: viewModel.confirmButtonCallToAction,
             showProcessingLabel: context.showProcessingLabel,
             compact: viewModel.shouldUseCompactConfirmButton,
-            linkAppearance: viewModel.linkAppearance
+            linkAppearance: viewModel.linkAppearance,
+            didTapWhenDisabled: { [weak self] in
+                self?.cardDetailsRecollectionSection.showAllValidationErrors()
+            }
```

Comment:
This shows validation errors in the recollection section if the user presses the disabled primary button.